src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.tsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { getCommaSeparatedNumber } from 'src/shared/helpers/formatters/numberFormatter';
+import {
+  getFeatureCoordinates,
+  getRegionName
+} from 'src/content/app/entity-viewer/shared/helpers/entity-helpers';
+
+import { Transcript } from 'src/content/app/entity-viewer/types/transcript';
+
+import transcriptsListStyles from '../DefaultTranscriptsList.scss';
+import styles from './TranscriptsListItemInfo.scss';
+
+type ItemInfoProps = {
+  transcript: Transcript;
+};
+
+const ItemInfo = (props: ItemInfoProps) => {
+  const { transcript } = props;
+
+  const getTranscriptLocation = () => {
+    const { start, end } = getFeatureCoordinates(transcript);
+    const startStr = getCommaSeparatedNumber(start);
+    const endStr = getCommaSeparatedNumber(end);
+    const regionName = getRegionName(transcript);
+    return `${regionName}:${startStr}-${endStr}`;
+  };
+
+  const getRNALength = () => {
```

Comment:
`getRNALength` -> `getSplicedRNALength`

(the current name may mean that you are getting the whole length of the transcript)

Reply:
✔️